ivy/functional/frontends/paddle/manipulation.py

```diff
@@ -77,25 +77,6 @@ def gather(params, indices, axis=-1, batch_dims=0, name=None):
     return ivy.gather(params, indices, axis=axis, batch_dims=batch_dims)
 
 
-@with_unsupported_dtypes(
-    {"2.5.1 and below": ("int8", "uint8", "int16", "uint16", "float16", "bfloat16")},
-    "paddle",
-)
-@to_ivy_arrays_and_back
-def gather_nd(x, index, name=None):
-    return ivy.gather_nd(x, index)
-
-
-@to_ivy_arrays_and_back
-def put_along_axis(arr, indices, values, axis, reduce="assign"):
-    result = ivy.put_along_axis(arr, indices, values, axis)
-    return result
-
-
-@with_supported_dtypes(
-    {"2.5.1 and below": ("int32", "int64", "float32", "float64")},
-    "paddle",
-)
```

Comment:
Hi @Dsantra92, the gather_nd function is the function I tried to add in manipulation.py file.  I am surprised why it's removed (and put_along_axi()). Sholld I add them back then push to remote branch again? Thanks.

Reply:
I run the following command below to view the differences between my working directory and the last committed version of the files: 

git diff ivy/... /manipulation.py ivy_tests/... /test_manipulation.py

<img width="824" alt="image" src="https://github.com/unifyai/ivy/assets/56084688/088fec37-8624-46a7-bffd-0d6cbed246e2">

I can add the mistakenly deleted function, and push it back?

Reply:
Yep, you can absolutely do that.

Reply:
<img width="533" alt="image" src="https://github.com/unifyai/ivy/assets/56084688/da0623f9-d4a8-4b47-aa4a-d49d2983e8ad">

<img width="415" alt="image" src="https://github.com/unifyai/ivy/assets/56084688/cb773208-0c24-4c2c-b9e0-4e02848fa3fb">


Hey @KareemMAX @kgmann, I just push these functions back pushed them as the pictures show. So, my question is, since there's already gather_nd function (in manipulation.py file) and its test function, there is no need to add it again, right?  Thank you!